src/app/components/ContactAdrField.js

```diff
@@ -5,8 +5,13 @@ import { c } from 'ttag';
 
 import { POST_BOX, EXTENDED, STREET, LOCALITY, REGION, POSTAL_CODE, COUNTRY } from '../constants';
 
+const initialAddress = (address) => {
+    const addressArray = Array.isArray(address) ? address : address.split(',');
+    return Array.from({ length: 7 }).map((_, i) => addressArray[i] || '');
```

Comment:
```suggestion
    return (new Array(7)).map((, i) => addressArray[i] || '');
```

Reply:
Not the same thing,

```js
> new Array(10).map((x, i) => i)
```
>[ <10 empty items> ]
```js
> Array.from({ length: 10}).map((x, i) => i)
```
>[ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 ]

Reply:
yeah, no undefined entries are needed; otherwise the ical library will complain later on.